tests/integration/components/polaris-drop-zone-test.js

```diff
@@ -1237,5 +1229,39 @@ module('Integration | Component | polaris-drop-zone', function(hooks) {
 
       assert.notOk(this.get('openFileDialog'));
     });
+
+    test('it supports passing `label` attributes', async function(assert) {
+      let actionFired = false;
+
+      this.setProperties({
+        actionFired,
+        labelAction: () => {
+          this.set('actionFired', true);
+        },
+      });
+
+      await render(hbs`
+        {{polaris-drop-zone
+          label="my label"
+          labelAction=(hash
+            text="my label action"
+            onAction=labelAction
+          )
+        }}
+      `);
+
+      assert
+        .dom(dropZoneLabelWrapperSelector)
+        .exists('A labelled component is wrapped around the dropzone');
+      assert.notOk(
```

Comment:
no harm here, though this is unnecessary....this is basically testing `polaris-labelled` component not firing the action on render....likely just that it fires on click should be enough

Reply:
Yeah I was just being super cautious about getting a false positive or anything. I'll remove it, this whole action test is likely not necessary since it should be covered in the labelled tests, but I'll at least keep the second assertion.

Reply:
sounds good to me!